README.md

```diff
@@ -710,11 +710,16 @@ For more information on the configuration parameter, see [Debugging/Profiling](h
 The driver currently does not support custom logging outside the usual logging frameworks like SLF4J. For more information on using SLF4J with the driver see [here](https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-logging-slf4j.html).
 
 ## Known Limitations
+Please note that as mentioned in the [maintenance policy](./RELEASE_POLICY.md), new features will not be added to the AWS JDBC Driver for MySQL. Future development will be taking place in the [AWS JDBC Driver](https://github.com/awslabs/aws-advanced-jdbc-wrapper).
```

Comment:
what about adding an extra line to invite people to take a look/migrate to the wrapper?

Please note that as mentioned in the [maintenance policy](./RELEASE_POLICY.md), there will not be any new feature work for the AWS JDBC Driver for MySQL. With the release of the [AWS JDBC Driver](link), all new feature development will be targeted towards AWS JDBC Driver. We encourage you to check the AWS JDBC Driver documentation(link) and to consider using it over AWS JDBC Driver for MySQL.